src/main/java/com/revature/model/Job.java

```diff
@@ -24,46 +24,40 @@
  */
 @Entity
 @Table(name = "jobs")
+

```

Comment:
No space needed between annotations and the thing being annotated.

Reply:
Changes are made as per suggestion